src/main/java/Game.java

```diff
@@ -3,10 +3,10 @@
 
 public class Game {
 
-    private Board board;
+    public Board board;
     public Player currentPlayer;
-    private Player player1;
-    private Player player2;
+    public Player player1;
+    public Player player2;
```

Comment:
I think I would keep these attributes private then in GamePlay call game.nextMove or similar, to get the next players move. This way you wouldn't need to expose the state in Game.